library/Icinga/Web/Navigation/ConfigMenu.php

```diff
@@ -0,0 +1,293 @@
+    const STATE_CRITICAL = 'critical';
+    const STATE_WARNING = 'warning';
+    const STATE_PENDING = 'pending';
+    const STATE_UNKNOWN = 'unknown';
+
+    protected $tag = 'ul';
+
+    protected $defaultAttributes = [
+        'class' => 'nav nav-level-1'
+    ];
+
+    protected $children = [
+        'system' => [
+            'title' => 'System',
+            'items' => [
+                'about' => [
+                    'label' => 'About',
+                    'url' => 'about'
+                ],
+                'health' => [
+                    'label' => 'Health',
+                    'url' => 'health',
+                ],
+                'announcements' => [
+                    'label' => 'Announcements',
+                    'url' => 'announcements'
+                ],
+                'sessions' => [
+                    'label' => 'User Sessions',
+                    'permission'  => 'application/sessions',
+                    'url'         => 'manage-user-devices'
+                ]
+            ]
+        ],
+        'configuration' => [
+            'title' => 'Configuration',
+            'permission' => 'config/*',
+            'items' => [
+                'application' => [
+                    'label' => 'Application',
+                    'url' => 'config/general'
+                ],
+                'authentication' => [
+                    'label' => 'Access Control',
+                    'permission'  => 'config/access-control/*',
+                    'url' => 'role/list'
+                ],
+                'navigation' => [
+                    'label' => 'Shared Navigation',
+                    'permission'  => 'config/navigation',
+                    'url' => 'navigation'
+                ],
+                'modules' => [
+                    'label' => 'Modules',
+                    'permission'  => 'config/modules',
+                    'url' => 'config/modules'
+                ]
+            ]
+        ],
+        'logout' => [
+            'items' => [
+                'logout' => [
+                    'label'       => 'Logout',
+                    'atts'  => [
+                        'target' => '_self',
+                        'class' => 'nav-item-logout'
+                    ],
+                    'url'         => 'authentication/logout'
+                ]
+            ]
+        ]
+    ];
+
+    protected $selected;
+
+    protected $cogItemActive = false;
+
+    protected function assembleUserMenuItem(HtmlElement $userMenuItem)
+    {
+        $username = Auth::getInstance()->getUser()->getUsername();
+
+        $userMenuItem->add(
+            HtmlElement::create(
+                'a',
+                Attributes::create(['href' => Url::fromPath('account')]),
+                [
+                    HtmlElement::create(
+                        'i',
+                        Attributes::create(['class' => 'user-ball']),
+                        Text::create($username[0])
```

Comment:
You're upper casing with CSS. Why not just directly here with `strtoupper`?

Reply:
it’s done the same way in the icingabweb .user-balls, so I adjusted to that.

Reply:
Hmm, well thinking about it a bit more.. This is locale specific. The server can do this locale aware, but only with locales we provide translations for, which are just a few. The browser on the other hand can probably do it with much more languages. So, leave it as is. It's fine, I guess.

Reply:
✅